content/content-management/types.md

```diff
@@ -48,7 +48,7 @@ The following examples take you stepwise through creating a new type layout for
 +++
 title = My First Event
 date = "2016-06-24T19:20:04-07:00"
-description = "Today is my 36th birthday. My how time flies."
+description = "Today is my 36th birthday. How time flies."
```

Comment:
This isn't a typo, haha. "My how time flies" is just kind of conversational. It's the equivalent of saying. "Today is my 36th birthday. Oh my! How time flies!" That said, I'm going to merge this anyways since it's better for our international audience. Thanks @digitalcraftsman